src/main/java/teammates/ui/controller/InstructorFeedbackAddAction.java

```diff
@@ -141,7 +141,8 @@ private void createTemplateFeedbackQuestions(String courseId, String feedbackSes
         return new ArrayList<FeedbackQuestionAttributes>();
     }
 
-    protected FeedbackSessionAttributes extractFeedbackSessionDataHelper(
+    @Override
+    protected FeedbackSessionAttributes extractFeedbackSessionDataSetUniqueAttributes(
```

Comment:
`setUniqueAttributesForSession`? Also, correct me if I'm wrong, but I don't think you need a return. And the argument should be `session` and not `newSession`.